### ISSUE TITLE
neutralize.sql:1: [W7930(file-not-used), ]  not used from manifest

### ISSUE THREAD

**Issue by yajo** [OCA/pylint-odoo#464]
## Module
See https://github.com/OCA/report-print-send/pull/328

## Describe the bug
pylint says there:

    base_report_to_printer/data/neutralize.sql:1: [W7930(file-not-used), ]  not used from manifest

However, it's expected to have `data/neutralize.sql` files in v16 that are not declared on manifests. That's a hardcoded path for the neutralization feature. See https://github.com/odoo/odoo/pull/101529 for history.

## To Reproduce
**Affected versions**: 16

1. Add a `data/neutralize.sql` file to a module

**Expected behavior**
Pylint happy

**Comment by moylop260** [comment]
I saw the CI green

Even I tried to run the following command:

```bash
git clone git@github.com:OCA/report-print-send.git -b 16.0 --depth=1 /tmp/report-print-send
pylint --load-plugins=pylint_odoo -d all -e file-not-used /tmp/report-print-send
```

And I even can not reproduce it

I'm using the following version:

```txt
pip freeze |grep pylint-odoo
pylint-odoo==8.0.9
```

Please, feel free to give more details how it can be reproduce it and re-open it
